package.json

```diff
@@ -28,7 +28,7 @@
     "build": "npm run clean && tsc",
     "format": "prettier -w .",
     "release": "npm run build && release-it",
-    "test": ""
+    "test": "node --test -r ts-node/register ./tests/**/*.test.ts"
```

Comment:
Are there any advantages of using node test vs jest? Maybe we'd want to set default test runners across the company and show by example. This is a question, I don't know which is better or better-suited.

Reply:
There are differences. One big preference that I have is using test runners that use native execution environments such as node's native test runner, ava, or tape.

Generally speaking, jest pulls a lot of unnecessary dependencies and is slower. Depending on the use case, extra features can be helpful or not.

Where do you find value in standardizing a test runner for unit and integration tests?

Reply:
The intent was to standardize the test runners across projects. But if native runners will perform better, then we should stick with those.
My idea was to use test runners which we would be able to recommend to people as a part of the CoP.

Reply:
Oh yeah, I got that part, but I'm not actively developing for some time so I was genuinely curious to hear your thoughts. Might be worth exploring.
Let's bring it up during some future CoP thread or sync.